.github/workflows/e2e.yaml

```diff
@@ -18,45 +19,25 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Check out code
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
         with:
           submodules: recursive
+      - uses: ./.github/actions/setup
```

Comment:
Nit: We could improve readability of the CI result page by giving this composite a name I think. What do you think?

<img width="321" alt="image" src="https://user-images.githubusercontent.com/1979133/205865860-e3fe2ddb-6ee3-4c84-baf9-11868d2944fc.png">

```suggestion
      - name: Setup Go and ko
        uses: ./.github/actions/setup
```